esm/walker.js

```diff
@@ -78,7 +78,11 @@ function parseAttributes(node, holes, parts, path) {
   var cache = new Map;
   var attributes = node.attributes;
   var remove = [];
-  var array = remove.slice.call(attributes, 0);
+  var html = parts.join('');
+  // This is a fix for returning an ordered list of attributes (IE and Edge don't)
+  var array = remove.slice.call(attributes, 0).sort(function(left, right) {
```

Comment:
I am not sure this works. Here an example:

```js
html`<p attr1=${1} attr2=${2} />`
```

the parts will be parsed as `<p attr1=` and then ` attr2=`, because that's the user input. If we change attributes order then one might not correspond to the developer intent.

In few words, it's forbidden to change the developer input intent, so I think this can break the logic.

Do you have an example that once shuffling attributes works as expected?